step-framework-collections/pom.xml

```diff
@@ -26,48 +26,43 @@
 		<dependency>
 			<groupId>commons-beanutils</groupId>
 			<artifactId>commons-beanutils</artifactId>
-			<version>1.9.4</version>
 		</dependency>
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-core</artifactId>
-			<version>${jackson.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.parsson</groupId>
```

Comment:
Why the new dependency? I did not see any new code/import .

Reply:
This is one of the things that prompted this entire "refactoring" in the first place - it's a new "hidden" dependency as a result of bumping the jackson version. It will compile, but won't run without parsson (or an alternative provider of whatever parsson does) because the loading is done via reflection using class names, and the compiled-in default is to look for this parsson thing. :)